src/index.html

```diff
@@ -76,11 +76,11 @@
             <span class="mobile-control-label">Mode</span>
             <ul class="radio-buttons">
                 <li>
-                    <input type="radio" name="mode" id="mode-dev" value="DEV" class="sends-mutate-request" checked>
+                    <input type="radio" data-property="mode" name="mode" id="mode-dev" value="DEV" class="sends-mutate-request">
```

Comment:
I assume this is no longer checked because main.js no sets the correct one to selected.

Reply:
Yep, main.js now sets the initial checked value based on the state provided by astmetad